### DISCUSSION TITLE
Please Add a TRM ON and OFF jumper on the future Blue SCSI DB25 V 2024 card

### DISCUSSION THREAD

**Discussion by lucg2** [BlueSCSI/BlueSCSI-v2#172] [Ideas]
I have an Atari TT 030. If I use a Desktop Blue SCSI V2 and an external DB 2025 Blue SCSI V2 I must have a TRM jumper on the last Card set to OFF otherwise the inside Desktop Blue SCSI isn't recognized by the Atari.
Please is further DB25 version a TRM jumper like the V 2022 had.
I tested with my Atari TT030 with the old DB 25 V 2022 which was perfect for my use. Unfortunately I broken this precious card....(SD card Slot).
Thank you.
Luc

**Comment by erichelgeson** [comment]
Hi there - would like to understand this a bit more - BlueSCSI DB25 is meant to be placed (and can really only be placed) at the end of the SCSI chain, which requires termination. What does your entire SCSI chain look like? I suspect there is some odd termination issue happening which is causing this issue, removing termination from the end of the chain doesn't seem like the "right" solution.

You can still build the 2022 version if you wish, the files still exist in github.

**Comment by lucg2** [comment]
Hello Eric

I made different tests with the last 2024 firmware and Atari TT030

BlueSCSI V2 Desktop and BlueSCSI V2 DB25 v2023 coexistence issues :

I use BlueSCSI V2 Desktop alone with an Atari TT030 TOS 3.06FR No issues in this case. But if i add a DB25 BlueSCSI for better SD card access and backup possibilities, i have issues.



Testing material :

BlueSCSI V2 Desktop50 pin revision 2023_10a  firmware v2024.04.02  jumper TRM ON  PWR ON  berg to molex cable with SD card image disk  HD10_512 20Mb.hda   

BlueSCSI V2 DB25 revision 2023.06a  firmware  v2024.04.02 with image disk  HD20_512 20Mb.hda   

 The result :

Atari + BlueSCSI V2 Desktop50 alone(SCSI 1.0) – Good

Atari + BlueSCSI V2 Desktop50 (SCSI 1.0) + DB25 external Boxed Hard drive(SCSI 3.0) – Good

Atari + 50 pin internal Hard Drive (SCSI 0.0) + BlueSCSI V2 DB25 (SCSI 2.0) – Good

Atari + BlueSCSI V2 Desktop50 (SCSI not seen) + BlueSCSI V2 DB25 (SCSI 2.0) – Only external BlueSCSI DB25 is seen

In this last case BlueSCSI Desktop is completely inaccessible and doesn't show up only the external BlueSCSI is seen and accessible (BUS 1 : SCSI 2.0).

 

But…..

Atari + BlueSCSI V2 Desktop50  (SCSI 1.0) “TERM OFF” + BlueSCSI V2 DB25 (SCSI 2.0) -Good : but it is not practical if DB 25 is removed, the Blue Desktop no longer boots (because TERM is OFF)



And now some test with the Old (v2022) and the new v2023 (jumper free)DB25 Bluescsi V2



Inside ATARI	Jumper TERM	SCSI ID	And	Outside Atari	Jumper TERM	SCSI ID	Result
BlueSCSI Desktop	ON	1.0	+	Nothing	 	 	OK
BlueSCSI Desktop	ON	1.0	+	Real Hard drive ext	Bouchon	3.0	OK
Real Hard Drive	resistor	0.0	+	BlueSCSI DB25 v2023	no jumper	2.0	OK
BlueSCSI Desktop	ON	not seen	+	BlueSCSI DB25 v2023	no jumper	2.0	only DB25 ok
BlueSCSI Desktop	OFF	1.0	+	BlueSCSI DB25 v2023	no jumper	2.0	OK
BlueSCSI Desktop	ON	1.0	+	BlueSCSI DB25 v2022	OFF	2.0	OK
BlueSCSI Desktop	ON	not seen	 	BlueSCSI DB25 v2022	ON	2.0	only DB25 ok
nothing	 	 	+	BlueSCSI DB25 v2022	ON	2.0	OK
nothing	 	 	+	BlueSCSI DB25 v2022	OFF	not seen	nothing


Thank you.

That’s why I calling for the return of the TERM jumper eventually future version of DB25 cards.

I bought my cards  by : https://shop.onegeekarmy.eu/index.php/product/bluescsi-v2-db25-smaller-design/

Blue SCSI. Is a very great thing for old computers.
Even for Atari MegaSTE and Atari TT

Thank you for your great job

Best regards,

Luc GANTNER :  France

**Comment by fdanapfel** [comment]
@lucg2 Do you have the 3 internal terminator resistor packs in your Atari TT030 removed (see https://mikrosk.github.io/doitarchive/doit_st/0d09.htm)?

If not then it is only possible to either use internal OR external SCSI devices, because the TT itself will be terminating one end of the SCSI chain.

Another thing that might help is to install the Term power mod on the TT mainboard: https://www.exxosforum.co.uk/forum/viewtopic.php?t=4933
